cmd/fossa/cmd/test/test.go

```diff
@@ -18,8 +18,20 @@ import (
 	"github.com/fossas/fossa-cli/config"
 )
 
+const defaultTestTemplate = `Test Failed. {{.Count}} {{if gt .Count 1 -}} issues {{- else -}} issue {{- end}} found:
```

Comment:
Here we say "Test Failed." but upon failure we say "Test passed!"

I think the messages should be more consistent at least with capitalization of the second word, e.g.: "Test failed" and "Test passed" — or capitalize second word in both cases.